en/user_guide/media.rst

```diff
@@ -0,0 +1,60 @@
+Media Management
+================
+
+Open Orchestra allows to manage media in the media library panel
+
+Folder
+------
+
+All files are organised inside folders. It's possible to browse the folder tree to see files by clicking on the left panel.
+
+Create
+~~~~~~
+
+Creating a folder is done by clicking on "new folder" button in the left panel.
+Creation form need:
+* name: name of the folder
+* list of websites: list of websites using the folder content (multiple select with CTRL + Click)
+
+Remove
+~~~~~~
+
+To remove a folder, go in a folder and click on the trash button on the right.
+
+Files
+-----
+
+Manage files
+~~~~~~~~~~~~
+
+The add button in the folder's content allows to upload files.
+
+A trash icon appears on media preview hovering to deleting media.
+
+Click on media preview open a modal to crop files or add meta information.
```

Comment:
A click ? opens ? modal box/window ?
seems unclear : if i click a media, a page opens, not a modal window. Then i have two tabs to switch between cropping and meta data.. I think this should be rephrased for better understanding